bbconf/schemas/bedfiles_schema.yaml

```diff
@@ -66,31 +68,43 @@ promotercore_percentage:
   description: Promoter core percentage
 tssdist:
   type: image
+  label: TSS_distance
   description: Region-TSS distance plot
 chrombins:
   type: image
+  label: Chromosome_bins
   description: Regions distribution over chromosomes plot
 gccontent:
   type: image
+  label: GC_content
   description: GC content plot
 paritions:
   type: image
+  label: Regions_dist_partitions
   description: Regions distribution over genomic partitions plot
 expected_partitions:
   type: image
+  label: Expected_partitions
   description: Expected distribution over genomic partitions plot
 cumulative_partitions:
   type: image
+  label: Cumulative_partitions
   description: Cumulative distribution over genomic partitions plot
 widths_histogram:
   type: image
+  label: Widths_histogram 
   description: Quantile-trimmed histogram of widths
 neighbor_distances:
   type: image
+  label: Neighbor_distance
   description: Distance between neighbor regions
 open_chromatin:
   type: image
+  label: Open_chromatin
   description: Cell specific enrichment for open chromatin plot
+genome:
+  type: object
```

Comment:
why is this an object, not a string?

Reply:
> why is this an object, not a string?

it should be string for this version. I'll change it back.

Reply:
> why is this an object, not a string?

for the current database, there is no  `genome` field in the bedfile table yet. the update make on `bedstats`  and `bedbuncher` added/changed the `genome` field with the refgenie digest, something like
```
genome: {
     alias: hg38
     digest: 2230c535660fb4774114bfa966a62f823fdb6d21acf138d4
}
```

should remove https://github.com/databio/bbconf/blob/539126e288a1f7d0dfd8e211d1492c2546e8af39/bbconf/schemas/bedfiles_schema.yaml#L105-L107 for this bbconf release?

Reply:
well, the point of this release is to make the prematurely released bedhost work, so set the genome type to what's needed for metadata served with the main instance of bedbase